lib/tools/transformer/options.dart

```diff
@@ -27,14 +27,19 @@ class TransformOptions {
    */
   final di.TransformOptions diOptions;
 
+  /// Option to generate the template cache.
+  final bool generateTemplateCache;
+
   TransformOptions({String sdkDirectory, List<String> htmlFiles,
       Map<String, String> templateUriRewrites,
-      di.TransformOptions diOptions}) :
+      di.TransformOptions diOptions, bool generateTemplateCache}) :
       sdkDirectory = sdkDirectory,
       htmlFiles = htmlFiles != null ? htmlFiles : [],
       templateUriRewrites = templateUriRewrites != null ?
           templateUriRewrites : {},
-      diOptions = diOptions {
+      diOptions = diOptions,
+      generateTemplateCache = generateTemplateCache != null ?
+          generateTemplateCache : false {
```

Comment:
Just to check, this line means that your change is noop, until someone adds `generate_template_cache: true` to their angular transformer in their `pubspec.yaml`? That means your change is not breaking and just adding a new functionality.